cypress/integration/my-orcid-sidebar.e2e.js

```diff
@@ -0,0 +1,119 @@
+describe('My Orcid sidebar' + runInfo(), () => {
+  describe('Orcid id', () => {
+    it('display the users Orcid', () => {})
+    it('display url to navigate to the public page view', () => {})
+  })
+  describe('Emails' + runInfo(), () => {
+    it('display a user with only a primary email "unverified"', () => {})
+    it('display a user with only a primary email "verified"', () => {})
+    it('add additional emails and display those as "unverified" with default privacy', () => {
+      // Expect changes to be display outside and inside of the modal
+      // TODO @dandineen no sketch that displays the input box
+    })
+    it('remove/delete  emails and display those as "unverified" with default privacy', () => {
+      // Expect changes to be display outside and inside of the modal
+    })
+    it('display emails "unverified" and "verified"', () => {
+      // Expect changes to be display outside and inside of the modal
+    })
+    it('resend the email verification email', () => {
+      // TODO @dandineen no button to resend
+    })
+    it('set a "checked" email as primary', () => {
+      // checks that a secondary email does not have the `make primary` button
+      // Expect changes to be display outside and inside of the modal
+    })
+    it('change the email privacy', () => {})
+    it('make changes and cancel', () => {
+      // Expect changes NOT to be display outside and inside of the modal
+    })
+    it('clicks outside and DONT close the modal ', () => {})
+    it('open the terms of use on a separate window ', () => {})
+  })
+  describe('Websites and socials links', () => {
+    it('display a user with no items', () => {
+      // TODO @dandineen should this look just empty on the modal
```

Comment:
I've added designs and example copy for the 'empty' lists in the modal forms -> https://zpl.io/b6qnERg

It has raised a question about the possibility of changing the 'Add another X' link text when the list is empty. In that case we really want the link to read `Add an X` instead. Is that something that can be done without too much difficulty?

Reply:
There would be no problem, we can change the string in the button depending on any condition. Please just update that as you wish on the designs